Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/Visuals/BoundsControlScaleHandles.cs

```diff
@@ -6,11 +6,11 @@
 
 namespace Microsoft.MixedReality.Toolkit.UI.Experimental
 {
-    [Serializable]
     /// <summary>
     /// Scale handles for <see cref="BoundsControl"/> that are used for scaling the
     /// gameobject BoundsControl is attached to with near or far interaction
     /// </summary>
+    [CreateAssetMenu(fileName = "BoundsControlScaleHandles", menuName = "Mixed Reality Toolkit/Bounds Control/Scale Handles")]
```

Comment:
Consideration: 

Should this be Scale Handle Config or Scale Handles Profile?

Thinking if am a new MRTK users, I right click and see create Scale Handles? Would I assume it will create like a prefab and geometry...or a definition file?

Reply:
renamed all of them to configuration for now :)